plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftUnload.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.redshift;
+
+import io.trino.operator.OperatorStats;
+import io.trino.operator.SplitOperatorInfo;
+import io.trino.testing.AbstractTestQueries;
+import io.trino.testing.QueryRunner;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.parallel.Execution;
+
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.collect.MoreCollectors.onlyElement;
+import static io.trino.plugin.redshift.RedshiftQueryRunner.AWS_ACCESS_KEY;
+import static io.trino.plugin.redshift.RedshiftQueryRunner.AWS_REGION;
+import static io.trino.plugin.redshift.RedshiftQueryRunner.AWS_SECRET_KEY;
+import static io.trino.plugin.redshift.RedshiftQueryRunner.IAM_ROLE;
+import static io.trino.plugin.redshift.RedshiftQueryRunner.S3_UNLOAD_ROOT;
+import static io.trino.tpch.TpchTable.NATION;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
+import static org.junit.jupiter.api.parallel.ExecutionMode.CONCURRENT;
+
+@TestInstance(PER_CLASS)
+@Execution(CONCURRENT)
+final class TestRedshiftUnload
+        extends AbstractTestQueries
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        return RedshiftQueryRunner.builder()
+                .setConnectorProperties(
+                        Map.of(
+                                "redshift.unload-location", S3_UNLOAD_ROOT,
+                                "redshift.unload-iam-role", IAM_ROLE,
+                                "s3.region", AWS_REGION,
+                                "s3.endpoint", "https://s3.%s.amazonaws.com".formatted(AWS_REGION),
+                                "s3.aws-access-key", AWS_ACCESS_KEY,
+                                "s3.aws-secret-key", AWS_SECRET_KEY,
+                                "s3.path-style-access", "true",
+                                "join-pushdown.enabled", "true",
+                                "join-pushdown.strategy", "EAGER"))
```

Comment:
Why do we need to set JOIN config properties?

Reply:
not required. Removed.